README.md

```diff
@@ -31,25 +31,27 @@ Users can call a high level `generate()` method, or run each iteration of the mo
 
 ## Coming very soon
 
-* Support for the Whisper model architectures
 * Support for DirectML
+* Support for the encoder decoder model architectures, such as whisper, T5 and BART.
+
+## Coming soon
+
+* Support for mobile devices (Android and iOS) with Java and Objective-C bindings
```

Comment:
optional: Shall we avoid distinguishing between coming soon and coming very soon?